toolkits/springernature/packages/springernature-header/scss/10-settings/colours.scss

```diff
@@ -0,0 +1,3 @@
+$global-colors: (
```

Comment:
This color already comes with the [springernature `brand-context`](https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/springernature/scss/10-settings/colors/default.scss#L8)